cli/user.go

```diff
@@ -46,13 +42,12 @@ func runGetUser(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	admin := client.NewAdminClient(&context.Context, context.Addr, client.User)
-	body, err := admin.GetYAML(args[0])
+	db := makeSQLClient()
+	err := processOneLine(db, fmt.Sprintf(`SELECT * FROM system.users WHERE username='%s'`, args[0]))
```

Comment:
We should use placeholders and parameters here instead of constructing queries with Sprintf (or use squalor?). SQL injection via the CLI is sort of a theoretical concern, but this is important enough that we should always be in the habit of doing it right.